File: Python/BotKicker.py
from flask import Flask
from flask import request
import requests
import json

app = Flask(__name__)

token = "Replace with your own token!"
urlStart = "https://api.telegram.org/bot" + token + "/"

@app.route('/BotKicker', methods = ['POST'])
def bot_kicker():
    update = json.loads(request.get_data(cache=False))
    adder_kicked = False

    if 'message' in update:
        message = update['message']
        chat_id = message['chat']['id']
        if 'new_chat_members' in message:
            new_chat_members = message['new_chat_members']
            for member in new_chat_members:
                if member['is_bot']:
                    bot_id = member['id']
                    chat_id = message['chat']['id']
                    if not adder_kicked:
                        adder_id = message['from']['id']
                        payload = {'chat_id': chat_id, 'user_id': adder_id}
                        requests.get(urlStart + "kickChatMember", params=payload)
                        adder_kicked = True
                    payload = {'chat_id': chat_id, 'user_id': bot_id}
                    requests.get(urlStart + "kickChatMember", params=payload)
    return json.dumps({'success':True}), 200, {'ContentType':'application/json'}

File: BotKicker.php
<?php
	//code
	$token = "Replace with your bot token!";
	$update = urldecode(file_get_contents('php://input'));
	$update = json_decode($update, true);
	$adder_kicked = false;
	
	if(array_key_exists('message', $update)) {
		$message = $update['message'];
		$message_id = $message['message_id'];
		if(array_key_exists('new_chat_members', $message)) {		
			$new_chat_members = $message['new_chat_members'];
			for ($i = 0; $i < sizeof($new_chat_members); $i++) {
				$member = $new_chat_members[$i];
				if($member['is_bot']) {
					$bot_id = $member['id'];
					$ch = curl_init();    					
					$chat_id = $message['chat']['id'];
					if($adder_kicked == false) {
						$adder_id = $message['from']['id'];
						curl_setopt($ch, CURLOPT_URL, "https://api.telegram.org/bot$token/kickChatMember?chat_id=$chat_id&user_id=$adder_id");
						curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
						curl_exec($ch);		    
						$adder_kicked = true;
					}
					curl_setopt($ch, CURLOPT_URL, "https://api.telegram.org/bot$token/kickChatMember?chat_id=$chat_id&user_id=$bot_id");
					curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
					curl_exec($ch);		
				}
			}
		}
	}	
?>
